### PR TITLE
Jugs are destructible

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR
This pr makes it so jugs are destructible.

## Why / Balance
Putting a jug in a backpack essentially made it indestructible to explosions. Nukies could carry half a dozen jugs of healing on them, and there would be zero counterplay to destroying their medical supplies. Considering beakers, vials, and bottles all broke, it makes sense to me to make jugs destructible as well.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Jugs are now destructible.

### CODE REVIEW

**Comment by deltanedas** [comment]
should be somewhat more durable imo since its plastic not brittle glass

gamer idea plastic lasts longer but as it takes damage leaks more and more :trollface:

File: Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml [hunk 1]
```diff
@@ -47,6 +47,28 @@
...
+              volume: -4
+        - !type:SpillBehavior { }
+        - !type:SpawnEntitiesBehavior
+          spawn:
+            SheetPlastic1:
+              min: 0
+              max: 1
+          transferForensics: true
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml:71]:
```suggestion
          acts: [ "Destruction" ]
```

File: Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml [hunk 2]
```diff
@@ -47,6 +47,28 @@
...
+          sound:
+            collection: MetalBreak
+            params:
+              volume: -4
+        - !type:SpillBehavior { }
+        - !type:SpawnEntitiesBehavior
+          spawn:
+            SheetPlastic1:
+              min: 0
+              max: 1
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml:68]: Something we do for nukes is just have a higher damager threshold that just deletes it so entities get cleaned up and it doesn't leave a bunch of debris everywhere.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
when bombing bed the jugs still empty their contents they just dont explode

**Comment by liltenhead** [comment]
Made them a bit stronger, and added the higher damage destruction so they disappear like other objects.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**